modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -170,28 +192,38 @@ definition {
 				thirdInvoice = "Document_1.png");
 		}
 
-		task ("And I go back to the MDF listing page") {
-			PRMUtils.gotoPRM();
+		task ("When Login with the Manager user and go to Partner Portal") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
 
-			PRMHomePage.gotoQuickLinksOption(
-				buttonLink = "mdf-requests",
-				buttonText = "MDF");
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
 		}
 
 		task ("And I go to MDF claim listing page") {
 			PRMNavigator.goToNavItem(navItem = "MDF Claim");
 		}
 
-		task ("When I export all visible MDF's") {
-			Button.click(button = "Export MDF Claim");
-		}
-
-		task ("Then the information about the visible MDF's will be inside the exported file") {
+		task ("And I get the IDs") {
 			var firstID = PRMMDFClaim.getClaimID(claimIndex = 1);
 			var secondID = PRMMDFClaim.getClaimID(claimIndex = 2);
+		}
+
+		task ("Then I search for the claim and check that the information is inside the file") {
+			PRMMDFClaim.searchClaimID(orderID = ${firstID});
+
+			Button.click(button = "Export MDF Claim");
 
 			PRMMDFRequest.viewExportedMDF(
-				fileContentList = "${firstID},${secondID}",
+				fileContentList = ${firstID},
+				fileName = "MDF Claim.csv");
```

Comment:
Hi @marinaasant0s, it should be inside a separated task 'Then the information about the searched Claim should be inside the downloaded file'